modules/services/emacs.nix

```diff
@@ -156,9 +161,17 @@ in {
           ExecStopPost =
             "${pkgs.coreutils}/bin/chmod --changes +w ${socketDir}";
         };
-      } // optionalAttrs (cfg.startWithUserSession) {
-        Install = { WantedBy = [ "default.target" ]; };
-      };
+      } // optionalAttrs (cfg.startWithUserSession == true
+        || cfg.startWithUserSession == "graphical") {
```

Comment:
```suggestion
      } // optionalAttrs (cfg.startWithUserSession != false) {
```

Reply:
I slightly prefer the origin code for better clarity but does not have a strong opinion.

Reply:
The original code seems easier to forget to extend the condition when new values are added to the enum (maybe irrelevant here, but as a general principle).